### ISSUE TITLE
Adapt to Angular2 rc6

### ISSUE THREAD

**Issue by JavOLeeN** [pleerock/ng2-radio-group#7]
Please adapt ng2-radio-group to Angular2 Release Candidate 6.

Errors:
node_modules/ng2-radio-group/CheckboxGroup.d.ts(1,10): error TS2305: Module '"../node_modules/@angular/common/index"' has no exported member 'Validator'.
node_modules/ng2-radio-group/CheckboxGroup.d.ts(1,21): error TS2305: Module '"../node_modules/@angular/common/index"' has no exported member 'ControlValueAccessor'.
node_modules/ng2-radio-group/CheckboxGroup.d.ts(1,43): error TS2305: Module '"../node_modules/@angular/common/index"' has no exported member 'Control'.
node_modules/ng2-radio-group/RadioGroup.d.ts(2,10): error TS2305: Module '"../node_modules/@angular/common/index"' has no exported member 'Validator'.
node_modules/ng2-radio-group/RadioGroup.d.ts(2,21): error TS2305: Module '"../node_modules/@angular/common/index"' has no exported member 'ControlValueAccessor'.
node_modules/ng2-radio-group/RadioGroup.d.ts(2,43): error TS2305: Module '"../node_modules/@angular/common/index"' has no exported member 'Control'.

Thanks!

**Comment by pleerock** [comment]
Im going to deprecate ng2-radio-group in favour of `ng2-select-controls`. try to install it and try it. [samples are here](https://github.com/pleerock/ng2-radio-group/tree/master/sample)

**Comment by mikesnare** [comment]
Any more info you can give on this?  ng-select-controls is referenced in NPM, but it points to a github repo that 404s (is it private)?

I'm upgrading from angular RC4 to GA and this is a blocker.  Any help would be appreciated.

**Comment by pleerock** [comment]
@mikesnare as I told ng2-radio-group is deprecated in favour of ng2-select-controls. can you try to install it and try it instead?

**Comment by mikesnare** [comment]
@pleerock yup.  The question was really more about the fact that ng-select-controls points to https://github.com/pleerock/ng2-select-controls according to npm and that repo is a 404.

I'm able to install the package via npm but it would be nice if there were an accessible repo for it.

Thanks!

**Comment by pleerock** [comment]
ah okay, I'll rename this repo

**Comment by mikesnare** [comment]
That was quick.  Thanks! 👍
